src/components/Modal/VotingPowerDelegationDetail/CandidateDetails.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+
+import Paragraph from 'decentraland-gatsby/dist/components/Text/Paragraph'
+import TokenList from 'decentraland-gatsby/dist/utils/dom/TokenList'
+import { Header } from 'decentraland-ui/dist/components/Header/Header'
+
+import LinkIcon from '../../Icon/LinkIcon'
+
+import './CandidateDetails.css'
+
+interface Props {
+  title: string
+  content?: string
+  links?: string[]
+  skills?: string[]
+}
+
+function linkShortener(link: string) {
```

Comment:
You can do this with css https://developer.mozilla.org/es/docs/Web/CSS/text-overflow